libs/ui/hover-card/brain/src/lib/brn-hover-card-content.service.ts

```diff
@@ -211,40 +229,45 @@ export class BrnHoverCardTriggerDirective implements OnInit, OnDestroy {
 		tap((visible) => visible && this._contentService.setState('open')),
 		switchMap((visible) => {
 			// we are delaying based on the configure-able input
-			return of(visible).pipe(delay(visible ? this.showDelay : this.hideDelay));
+			return of(visible).pipe(delay(visible ? this.showDelay() : this.hideDelay()));
 		}),
 		switchMap((visible) => {
 			// don't do anything when we are in the process of showing the content
 			if (visible) return of(visible);
 			// we set the state to closed here to trigger any animations for the element leaving
 			this._contentService.setState('closed');
 			// then delay to wait for the leaving animation to finish
-			return of(visible).pipe(delay(this.animationDelay));
+			return of(visible).pipe(delay(this.animationDelay()));
 		}),
 		distinctUntilChanged(),
 		share(),
 		takeUntil(this._destroy$),
 	);
 
-	@Input()
-	public showDelay = 300;
-	@Input()
-	public hideDelay = 500;
-	@Input()
-	public animationDelay = 100;
-	@Input()
-	public sideOffset = 5;
-
-	@Input()
-	public align: 'top' | 'bottom' = 'bottom';
+	public showDelay = input(300);
+	public hideDelay = input(500);
+	public animationDelay = input(100);
+	public sideOffset = input(5);
+	public align = input<'top' | 'bottom'>('bottom');
 
-	@Input()
-	public set brnHoverCardTriggerFor(value: TemplateRef<unknown> | BrnHoverCardContentDirective) {
-		this._contentService.setContent(value, this._vcr);
-	}
+	public readonly brnHoverCardTriggerForInput = input<TemplateRef<unknown> | BrnHoverCardContentDirective | undefined>(
+		undefined,
+		{ alias: 'brnHoverCardTriggerFor' },
+	);
+	public readonly brnHoverCardTriggerForState = computed(() => signal(this.brnHoverCardTriggerForInput()));
+	private readonly _brnHoverCardTriggerFor = computed(() => this.brnHoverCardTriggerForState()());
+	private readonly _brnHoverCardTriggerForEffect = effect(
```

Comment:
In this repo we usually define the effects in the constructor instead of creating a unused variable.